config/environments/development.rb

```diff
@@ -85,4 +85,12 @@
                               }
                             )
 
+
+  config.after_initialize do
```

Comment:
You still have to enable it in your [config](https://github.com/openSUSE/osem/blob/master/config/config.yml.example#L22) but when you do it is straight in test mode. No need to edit the [initializer](https://github.com/openSUSE/osem/blob/master/config/initializers/devise.rb#L270) anymore.